src/main/java/teammates/ui/webapi/action/GetFeedbackSessionsAction.java

```diff
@@ -20,12 +23,25 @@ protected AuthType getMinAuthLevel() {
 
     @Override
     public void checkSpecificAccessControl() {
```

Comment:
Actually, you don't need have too many if here.
```java
if (IS_IN_RECYCLE_BIN == null) {
    if (courseId = null) {
        assert userInfo.isStudent
    } else {
        assert student can access course
    }
} else {
    if (courseId = null) {
        assert userInfo.isInstructor
    } else {
        assert instructor can access course
    }
}
```

Having `userInfo.isInstructor && ! isRequestAsStudent` is confusing especially for new contributor (if you don't explain the situation in the comment)

Reply:
I have looked at the logic again and finds that it indeed can be simplified.  Especially `userInfo.isInstructor` is no longer necessary to check in action since it already passed accessControl.